### ISSUE TITLE
Accessibility: Disabled menu order buttons focusable

### ISSUE THREAD

**Issue by Cheffheid** [voldemortensen/menu-customizer#78]
The spans that control a menu item's order are focusable regardless of whether they can be activated or not. 

This adds tedium for a sighted person when tabbing through(since they can sort of tell that they are focusing the disabled option). However, when using a screen reader it becomes confusing since the screen reader will announce it as a clickable element, even though it doesn't actually do anything.

**Comment by celloexpressions** [comment]
This one would not be very elegant to fix, but we should be able to just toggle the tabindex whenever we toggle the enabled classes here. Some of the disabled styling is handled automatically with CSS currently though, so this may end up being a bit of a rabbit hole to try fixing.

**Comment by celloexpressions** [comment]
Best option would probably to add a disabled attribute to buttons that should be disabled and refresh these when items are added or removed. Especially now that they're actual buttons. That'll also allow us to clean up the associated CSS selectors, rather than trying to rely on :first-item, etc.

**Comment by celloexpressions** [comment]
This is not an easy thing to fix - we may not be able to fix it at all but will hopefully figure something out. Needs to wait for now at least.

**Comment by ocean90** [comment]
Seems to be fixed with https://github.com/voldemortensen/menu-customizer/pull/112/.
